deployment/docker-compose.yml

```diff
@@ -64,6 +64,18 @@ services:
       - ./data/certbot/conf:/etc/letsencrypt
       - ./data/certbot/www:/var/www/certbot
     entrypoint: "/bin/sh -c 'trap exit TERM; while :; do certbot renew; sleep 12h & wait $${!}; done;'"
+  alignScore:
+    image: idinsight/aaq-backend:latest
+    profiles:
```

Comment:
This profile means this container will only be started up if you use `docker compose` with the `--profile` flag. I'll add this to docs.

See: https://docs.docker.com/compose/profiles